lib/iris/fileformats/netcdf/saver.py

```diff
@@ -2083,8 +2095,32 @@ def add_ellipsoid(ellipsoid):
 
                 self._coord_systems.append(cs)
 
-            # Refer to grid var
-            _setncattr(cf_var_cube, "grid_mapping", cs.grid_mapping_name)
+            # create grid mapping string:
+            coords = cube.coords(coord_system=cs)
+            # TODO: How do we sort these coords?
+            #       For DimCoords, we can use the dimension order, but what
+            #       about AuxCoords that could be 2D?
+
+            # prefer netCDF variable name, if exists, else default ro coord.name()
+            coord_string = " ".join(
+                [coord.var_name if coord.var_name else coord.name() for coord in coords]
+            )
+            grid_mappings.append((cs.grid_mapping_name, coord_string))
+
+        # Refer to grid var
+        # TODO: Future flag for extended grid_mapping syntax?
```

Comment:
Yes, that was an agreed part of the spec 👍

Reply:
Added as part of 8fd4f0c9

Reply:
Covered by new ordered_axis propert on Cube object.